public/components/trace_analytics/requests/queries/traces_queries.ts

```diff
@@ -568,7 +568,7 @@ export const getCustomIndicesTracesQuery = (
         must_not: [],
       },
     },
-    ...(sort && { sort: [{ [sort.field]: { order: sort.direction } }] }),
+    ...(sort ? { sort: [{ [sort.field]: { order: sort.direction } }] } : {}),
```

Comment:
is this change needed?

Reply:
TBH not needed, I was seeing some issues with search made with keyboard hit `shift+enter`. The root cause was this: `onSearch={(searchQuery) => props.refresh(undefined, searchQuery)}`. But this is part of experimental feature and will be changed when it becomes GA.

Reply:
reverted 😄